apps/e2e/forms-storybook/src/app/single-file-attachment/single-file-attachment.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, Input } from '@angular/core';
+import {
+  AbstractControl,
+  UntypedFormBuilder,
+  UntypedFormControl,
+  UntypedFormGroup,
+  Validators,
+} from '@angular/forms';
+import { SkyFileAttachmentChange, SkyFileItem } from '@skyux/forms';
+
+@Component({
+  selector: 'app-single-file-attachment',
+  templateUrl: './single-file-attachment.component.html',
+  styleUrls: ['./single-file-attachment.component.scss'],
+})
+export class SingleFileAttachmentComponent {
+  @Input()
+  public set uploadedFiles(value: SkyFileItem) {
+    this.reactiveFileUpdated({ file: value });
+    this.showStatesFlag = false;
+  }
+
+  public showStatesFlag = true;
```

Comment:
is this variable doing anything? it looks like it gets set to false but never used anywhere